pkgs/tools/security/dkimpy/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, pythonPackages, fetchurl, openssl, makeWrapper }:
```

Comment:
@Mic92 the package is not defined outside of `pythonPackages` though, it is `callPackage`d from `python-packages.nix`.